tools/make-deb.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+#
+# Produce a .deb from a built Boulder plus helper files.
+#
+# This script expects to run on Ubuntu, as configured on GitHub Actions runners
+# (with curl, make, and git installed).
+#
+# -e Stops execution in the instance of a command or pipeline error.
+# -u Treat unset variables as an error and exit immediately.
+set -eu
+cd "$(realpath -- $(dirname -- "$0"))/.."
```

Comment:
I think the mixed quoting here means a pathname containing spaces would be split before hitting `realpath`.

```suggestion
cd "$(realpath -- "$(dirname -- "$0")")/.."
```